LFGBulletinBoard/LfgToolList.lua

```diff
@@ -643,6 +648,8 @@ function GBB.LfgClickRequest(self,button)
 		if IsShiftKeyDown() then
 			WhoRequest(req.name)
 			--SendWho( req.name )
+		elseif IsAltKeyDown() then
```

Comment:
Done. Leaving the alt+click behavior on RequestList as well, just in case you'd prefer messaging without requesting invite. It affects none of the functionality, obviously.